session.go

```diff
@@ -43,3 +45,14 @@ func (s *Session) FlushStorage() (err error) {
 	_, err = synchronousEvent(s.ctx, s, s.w, Event{Name: EventNameSessionCmdFlushStorage, TargetID: s.id}, EventNameSessionEventFlushedStorage)
 	return
 }
+
+// Loads a chrome extension
+func (s *Session) LoadExtension(path string) (err error) {
+	Path string `json:"path"`
```

Comment:
You need to add this line [below Password](https://github.com/asticode/go-astilectron/blob/master/event.go#L41) instead